webservices/legal_docs/archived_murs.py

```diff
@@ -180,7 +180,7 @@ def get_single_mur(mur_no):
                 "doc_id": "mur_{0}".format(row["mur_no"]),
                 "no": row["mur_no"],
                 "case_serial": row["mur_id"],
-                "url": "/legal/matter-under-review/{0}/".format(row["mur_no"]),
+                "url": "/legal/matter-under-review/{0}/".format(row["mur_no"]),  # Do we need this?
```

Comment:
Oh interesting, I checked the CMS app and we don't use this. I don't think we'll ever need it either as it's the single MUR page already and having it here would just be redundant.